server/src/main/java/org/apache/druid/guice/http/JettyHttpClientModule.java

```diff
@@ -91,7 +91,7 @@ public HttpClient get()
       httpClient.setIdleTimeout(config.getReadTimeout().getMillis());
       httpClient.setMaxConnectionsPerDestination(config.getNumConnections());
       httpClient.setMaxRequestsQueuedPerDestination(config.getNumRequestsQueued());
-      httpClient.setConnectTimeout(CLIENT_CONNECT_TIMEOUT_MILLIS);
+      httpClient.setConnectTimeout(config.getClientConnectTimeout());
```

Comment:
It would be better to remove `CLIENT_CONNECT_TIMEOUT_MILLIS` variable as well

Reply:
+1